pkg/ottl/expression.go

```diff
@@ -245,6 +246,39 @@ func (g StandardFloatGetter[K]) Get(ctx context.Context, tCtx K) (float64, error
 	}
 }
 
+type FunctionGetter[K any] interface {
```

Comment:
Could you add Godoc comments for any of the new public interfaces or methods in this file?

Reply:
added these